hw/bsp/nordic_pca10095_net/bsp.yml

```diff
@@ -29,7 +29,7 @@ bsp.linkerscript.BOOT_LOADER.OVERWRITE:
     - "hw/bsp/nordic_pca10095_net/boot-nrf5340_net.ld"
     - "@apache-mynewt-core/hw/mcu/nordic/nrf5340_net/nrf5340_net.ld"
 bsp.part2linkerscript: "hw/bsp/nordic_pca10095_net/split-nordic_pca10095_net.ld"
-bsp.downloadscript: "hw/bsp/nordic_pca10095_net/nordic_pca10095_net_download.sh"
+bsp.downloadscript: "@apache-mynewt-core/hw/scripts/download.sh"
```

Comment:
Is the repo name needed in the location for this BSP for some reason?

Reply:
no, I will remove it for consistency

Reply:
done